client/views/chan.tpl

```diff
@@ -29,9 +29,16 @@
 	{{#notEqual type "lobby"}}
 		<span class="name" title="{{name}}">{{name}}</span>
 		<span class="badge{{#if highlight}} highlight{{/if}}" data-highlight="{{highlight}}">{{#if unread}}{{roundBadgeNumber unread}}{{/if}}</span>
-		<span class="close-tooltip tooltipped tooltipped-w" aria-label="Leave">
-			<button class="close" aria-label="Leave"></button>
-		</span>
+		{{#equal type "channel"}}
+			<span class="close-tooltip tooltipped tooltipped-w" aria-label="Leave">
+				<button class="close" aria-label="Leave"></button>
+			</span>
+		{{/equal}}
+		{{#notEqual type "channel"}}
```

Comment:
Mind replacing these 2 lines with `{{else}}` like we do in https://github.com/thelounge/thelounge/blob/master/client/views/image_viewer.tpl#L18 for example?
Feel free to do the same in L29/42 if you want (but don't have to).

Reply:
done